### PR TITLE
i18n(de): translate components/tabs

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `components/tabs.mdx` into German language.

### CODE REVIEW

File: docs/src/content/docs/de/components/tabs.mdx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+Um eine Oberfläche mit Registerkarten zu erstellen, verwende die Komponenten `<Tabs>` und `<TabItem>`.
+Registerkarten sind nützlich, um gleichwertige Informationen zu gruppieren, wenn ein Nutzer nur eine von mehreren Optionen sehen muss.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/tabs.mdx:17]: I think you translated these labels in a few other pages so I guess we could do the same here (multiple occurrences in this page).

File: docs/src/content/docs/de/components/tabs.mdx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+</Preview>
+
+### Registerkarten synchronisieren
+
+Halte mehrere Registerkartengruppen synchron, indem du das Attribut [`syncKey`](#synckey) hinzufügst.
+
+Alle `<Tabs>` auf einer Seite mit demselben Wert für `syncKey` zeigen dasselbe aktive Label an.
+
+Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
+
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/tabs.mdx:74]:
```suggestion
Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
```

To match the original version.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for another great translation :star2:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left a few suggestions but otherwise looks good to me.

File: docs/src/content/docs/de/components/tabs.mdx (hunk 2 above)
Comment (trueberryless) [reply docs/src/content/docs/de/components/tabs.mdx:74]: I'm sorry, but I don't understand what I need to change? May you please add an additional explanation maybe?

File: docs/src/content/docs/de/components/tabs.mdx [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+</Preview>
+
+### Registerkarten synchronisieren
+
+Halte mehrere Registerkartengruppen synchron, indem du das Attribut [`syncKey`](#synckey) hinzufügst.
+
+Alle `<Tabs>` auf einer Seite mit demselben Wert für `syncKey` zeigen dasselbe aktive Label an.
+
+Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/tabs.mdx:73]:
```suggestion
Alle `<Tabs>` auf einer Seite mit demselben Wert für `syncKey` zeigen dasselbe aktive Label an.
Auf diese Weise kann dein Leser eine Auswahl treffen (z. B. sein Betriebssystem oder den Paketmanager), die dann beim Navigieren durch die Seiten beibehalten wird.
```

File: docs/src/content/docs/de/components/tabs.mdx (hunk 2 above)
Comment (HiDeoo) [reply docs/src/content/docs/de/components/tabs.mdx:74]: Sorry, not sure what I did with this suggestion, I made another one (it's to remove an extra line) 😄

Comment (trueberryless) [reply docs/src/content/docs/de/components/tabs.mdx:73]: No problem, happens to everybody... Thank you for having such a good eye, though! 🎉

File: docs/src/content/docs/de/components/tabs.mdx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+Um eine Oberfläche mit Registerkarten zu erstellen, verwende die Komponenten `<Tabs>` und `<TabItem>`.
+Registerkarten sind nützlich, um gleichwertige Informationen zu gruppieren, wenn ein Nutzer nur eine von mehreren Optionen sehen muss.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/tabs.mdx:17]:
```suggestion
	<TabItem label="Sterne">Sirius, Wega, Betelgeuse</TabItem>
	<TabItem label="Monde">Io, Europa, Ganymed</TabItem>
```

One last change and we should be good to go 👍

**Comment by trueberryless** [comment]
By the way: Hugh thanks to you for reviewing all my PRs, we are close to having 100% for Starlight German translation again!

I'm hyped because it kinda feels like a 100% speedrun xD

**Review by HiDeoo — APPROVED** [review]
Thanks for the quick fixes.

> By the way: Hugh thanks to you for reviewing all my PRs, we are close to having 100% for Starlight German translation again!

Well, huge thanks to you for all the hard work and it's great to see German rising up the charts again. I'm not the fastest to review German and some PRs may require some help/validation from Chris, but it's getting there :rocket:

**Comment by trueberryless** [comment]
> Well, huge thanks to you for all the hard work and it's great to see German rising up the charts again. I'm not the fastest to review German and some PRs may require some help/validation from Chris, but it's getting there :rocket:

Yeah, no hurry, some breaks are also very important, especially at these times... I mean after midnight. At least for me it's better to take it slowly instead of rushing through it and making more mistakes on my way...

**Pull request merged**